website/layouts/partials/hooks/body-end.html

```diff
@@ -0,0 +1,58 @@
+<script>
+
+    //create the html element with the version string
+    function WarningElement(version) {
+        let RawElement = `
+        <div class="alert alert-warning" role="alert">
+            <h3>
+                You are viewing documentation for Karpenter version: ${version}
+            </h3>
+            <p> Karpenter ${version} documentation is not the latest stable release. The version you are currently viewing is a static snapshot. For up-to-date documentation, see the 
+                <a href="/docs/">latest version.</a>
+            </p>
+        </div>
+        `
+        let TemplateElement = document.createElement('div')
+        TemplateElement.innerHTML = RawElement
+
+        return TemplateElement
+    }
+
+    //create the html element with the version string
+    function PrereleaseElement() {
+        let RawElement = `
+        <div class="alert alert-warning" role="alert">
+            <h3>
+                You are viewing pre-release documentation for Karpenter 
```

Comment:
You can share this with the section above if you name the preview version something human readable.
```suggestion
                You are viewing documentation for Karpenter version: ${version}
```

Reply:
I'd like to keep it separate, because pre-release isn't a "static snapshot"